Code/C++/Brian_Kernighan.cpp

```diff
@@ -0,0 +1,65 @@
+// Brian Kernighan's Algorithm
```

Comment:
Include this in multiline comment itself.

Reply:
I have made the necessary changes as requested. Please let me know if any further changes are required or not.
Regards,
Debanjan Poddar
@raksha009 
@tarun26091999